Documentation/docs-mobile/messages/apt2000.md

```diff
@@ -0,0 +1,29 @@
+---
+title: .NET for Android error/warning APT2000
+description: APT2000 error/warning code
+ms.date: 04/11/2024
+---
+# .NET for Android error/warning APT2000
+
+## Well-known APT2000 error messages
+
+- `error APT2000: Invalid file name: must contain only [a-z0-9_.]`
+- `error APT2000: Resource entry resource_name is already defined.`
+- `error APT2000: No resource found that matches the given name (at 'resource_name' with value '@string/foo').`
+- `error APT2000: invalid resource directory name: obj\Debug\dir with spaces "dir with spaces".`
```

Comment:
Should this one be?

```suggestion
- `error APT2000: invalid resource directory name: "obj\Debug\dir with spaces".`
```

Reply:
The fact that we have these "well-known" error messages suggests that we should *update* the `error_codes` list so that these become *distinct* error codes, not just APT2000:
https://github.com/dotnet/android/blob/27feb809cde06489c800aad06305fe33f6a2f1c7/src/Xamarin.Android.Build.Tasks/Tasks/Aapt2.cs#L278

Reply:
Yes, but this does not invalidate the need for this PR and the message. That should be handled in a new PR which tackles specific messages, since we should be adding unit tests for all of those to make sure the correct message is raised.

Reply:
The wording really should be "Example" not "Well Known"